pkgs/by-name/qb/qbittorrent/package.nix

```diff
@@ -1,77 +1,66 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-
-, boost
-, cmake
-, Cocoa
-, libtorrent-rasterbar
-, ninja
-, qtbase
-, qtsvg
-, qttools
-, wrapGAppsHook3
-, wrapQtAppsHook
-
-, guiSupport ? true
-, dbus
-, qtwayland
-
-, trackerSearch ? true
-, python3
-
-, webuiSupport ? true
+{
+  apple-sdk_15,
+  boost,
+  cmake,
+  dbus,
+  fetchFromGitHub,
+  guiSupport ? true,
+  hicolor-icon-theme,
+  lib,
+  libtorrent-rasterbar,
+  openssl,
+  pkg-config,
+  python3,
+  qt6,
+  stdenv,
+  trackerSearch ? true,
+  webuiSupport ? true,
+  zlib,
 }:
```

Comment:
In a `pkgs/by-name` world, doesn't `lib` and `stdenv` still come first in the argument attrset?

I'd also tend to group the configuration knobs together, or in "islands" as was done originally.

Reply:
It doesn't really matter if lib and stdenv are first. This is more of a structure preference, me liking things in alphabetical order.